app/views/shared/_contact_panels.html.erb

```diff
@@ -113,111 +114,173 @@
 <% end %>
 
 <% if show_floating_chat? %>
-<a href="#" aria-hidden="true" id="js-chat-popup" class="chat-box__container chat-box__container--hidden t__floating-web-chat">
-    <div class="chat-box__heading">Web Chat</div>
-    <span class="icon icon--web-chat"></span>
+<a href="#" aria-hidden="true" id="js-chat-popup"
+  class="chat-box__container chat-box__container--hidden t__floating-web-chat">
+  <div class="chat-box__heading">Web Chat</div>
+  <span class="icon icon--web-chat"></span>
 </a>
 <% end %>
 
-<a href="#" aria-hidden="true" class="chat-popup__container" data-dough-component="ChatPopup">
+<div aria-hidden="true" class="mobile-webchat__container mobile-webchat--closed" data-dough-component="ChatPopup">
+  <button class="mobile-webchat__close">
+    <span>X</span>
+  </button>
+  <div class="mobile-webchat--icon">
     <span class="icon icon--web-chat"></span>
     <svg class="icon-svg--web-chat">
       <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-icon--web-chat"></use>
     </svg>
-</a>
+  </div>
+  <div class="mobile-webchat__form">
+    <h4><%= t('contact_panels.chat_popup.title') %></h4>
+    <div class="mobile-webchat__form-select--container">
```

Comment:
I think we should use a slightly different naming convention here and below: since this element is a descendent of ```mobile-webchat__form``` it should probably be marked up as ```mobile-webchat__form__select-container``` or maybe more succinctly as ```mobile-webchat__select-container```.